build/dependencies.props

```diff
@@ -1,8 +1,11 @@
 <Project>
   <PropertyGroup>
     <AspNetCoreVersion>2.0.0-*</AspNetCoreVersion>
+    <CoreFxVersion>4.3.0</CoreFxVersion>
```

Comment:
IIUC as a part of retargeting for ns2.0, we should start using the 4.4.0-\*. There may be some conflicts between corefx 4.3.0 and NS.Library 2.0.0